kotest-runner/kotest-runner-junit5/src/jvmMain/kotlin/io/kotest/runner/junit/platform/JUnitTestEngineListener.kt

```diff
@@ -171,10 +169,7 @@ class JUnitTestEngineListener(
    private fun markSpecStarted(kclass: KClass<*>): TestDescriptor {
       return try {
 
-         val descriptor = getSpecDescriptor(root, kclass.toDescriptor(), formatter.format(kclass))
-
-         logger.log { Pair(kclass.bestName(), "Registering dynamic spec $descriptor") }
-         listener.dynamicTestRegistered(descriptor)
```

Comment:
💬 Since these descriptors are now returned as part of discovery they are no longer dynamic and must not be registered.